docs/src/contrib.md

```diff
@@ -2,23 +2,31 @@
 
 If you are interested in contributing to this package,
 please consider going through this guide to help make your
-developing workflow smooth.
+development workflow as smooth as possible.
 
-Contributing to the documentation is always a good start to get yourself familiar
+Contributing to documentation and unit tests are always great ways to get yourself familiar
 with the community and workflows.
 
+## CLI Tool
+
+There is a CLI tool in this repository at `.ci/run` that can help
+you simplify your workflow substantially. You can run `.ci/run -h` in your
+terminal to see the help message. or run `.ci/run <command> -h`
+to see the help message of each command.
+Below are some common use cases.
+
 ## Documentation
 
 ## Setup Documentation
 
-If you are editing the documentation, you can use the `serve` command:
+If you're editing the documentation, you can use the `serve` command:
 
```

Comment:
in scientific writing, we usually do not write abbreviation, so I think "you are" might be better?

Reply:
Got it! Apologies on my end, I'll go ahead and undo the abbreviations